modules/claims_api/app/swagger/claims_api/forms/form_0966_response_swagger.rb

```diff
@@ -23,38 +23,70 @@ class Form0966ResponseSwagger
-              key :format, 'datetime'
-              key :example, '2014-07-28T19:53:45.810+00:00'
-              key :description, 'Datetime intent to file was first called'
-            end
+              property :received_date do
+                key :type, :string
+                key :format, 'datetime'
+                key :example, '2014-07-28T19:53:45.810+00:00'
+                key :description, I18n.t('claims_api.field_descriptions.received_date')
+              end
 
-            property :expiration_date do
-              key :type, :string
-              key :format, 'datetime'
-              key :example, '2015-08-28T19:52:25.601+00:00'
-              key :description, 'One year from initial intent to file Datetime'
-            end
+              property :expiration_date do
+                key :type, :string
+                key :format, 'datetime'
+                key :example, '2015-08-28T19:52:25.601+00:00'
+                key :description, 'One year from initial intent to file Datetime'
+                key :description, I18n.t('claims_api.field_descriptions.expiration_date')
+              end
 
-            property :type do
-              key :type, :string
-              key :example, 'compensation'
-              key :description, 'Type of claim being submitted'
-              key :enum, %w[
-                compensation
-                burial
-                pension
-              ]
-            end
+              property :type do
+                key :type, :string
+                key :example, 'compensation'
+                key :description, I18n.t('claims_api.field_descriptions.type')
+                key :enum, %w[
+                  compensation
+                  burial
+                  pension
+                ]
+              end
+
+              property :status do
+                key :type, :string
+                key :example, 'active'
+                key :description, I18n.t('claims_api.field_descriptions.status')
+              end
+
+              property :intent_to_file_id do
+                key :type, :integer
+                key :example, '234'
+                key :description, I18n.t('claims_api.field_descriptions.intent_to_file_id')
+              end
 
-            property :status do
-              key :type, :string
-              key :example, 'active'
-              key :description, 'Says if the Intent to File is Active or Expired'
+              property :participant_claimant_id do
+                key :type, :integer
+                key :example, '234'
+                key :description, I18n.t('claims_api.field_descriptions.participant_claimant_id')
+              end
+              property :participant_vet_id do
+                key :type, :integer
+                key :example, '234'
+                key :description, I18n.t('claims_api.field_descriptions.intent_to_file_id')
+              end
+              property :signature_indicated do
```

Comment:
I thought we removed this from the BGS call, do we still want to include it if we aren't sending it upstream anymore?

Reply:
this was been fixed